search-resources-api/src/test/java/no/unit/nva/dynamodb/DynamoDBStreamHandlerTest.java

```diff
@@ -53,321 +55,312 @@
+            MISSING_FIELD_LOGGER_WARNING_TEMPLATE.replace(PLACEHOLDER_LOGS, PLACEHOLDER_STRINGS);
+    public static final String WHITESPACE = "   ";
     private static final String SAMPLE_JSON_RESPONSE = "{}";
+
     private DynamoDBStreamHandler handler;
     private Context context;
     private Environment environment;
-
-    private JsonNode contributorTemplate;
     private TestAppender testAppender;
-    public static final String EXPECTED_MESSAGE = "expectedMessage";
+    private RestHighLevelClient restClient;
+    private SearchResponse searchResponse;
 
     /**
      * Set up test environment.
      *
-     * @throws IOException some error occurred
      */
     @BeforeEach
-    public void init() throws IOException {
+    void init() {
         context = mock(Context.class);
         environment = setupMockEnvironment();
+        restClient = mock(RestHighLevelClient.class);
+        searchResponse = mock(SearchResponse.class);
 
-        RestHighLevelClient restHighLevelClient = mock(RestHighLevelClient.class);
-        SearchResponse searchResponse = mock(SearchResponse.class);
-        DeleteResponse deleteResponse = mock(DeleteResponse.class);
-        when(searchResponse.toString()).thenReturn(SAMPLE_JSON_RESPONSE);
-        when(restHighLevelClient.search(any(), any())).thenReturn(searchResponse);
-        when(restHighLevelClient.delete(any(), any())).thenReturn(deleteResponse);
-
-        ElasticSearchHighLevelRestClient elasticSearchRestClient = new ElasticSearchHighLevelRestClient(environment,
-                restHighLevelClient);
-
-        testAppender = LogUtils.getTestingAppender(DynamoDBStreamHandler.class);
+        var elasticSearchRestClient = new ElasticSearchHighLevelRestClient(environment,
+                restClient);
         handler = new DynamoDBStreamHandler(elasticSearchRestClient);
-        contributorTemplate = mapper.readTree(IoUtils.inputStreamFromResources(Paths.get(CONTRIBUTOR_TEMPLATE_JSON)));
+        testAppender = LogUtils.getTestingAppender(DynamoDBStreamHandler.class);
     }
 
     @Test
-    @DisplayName("testCreateHandlerWithEmptyEnvironmentShouldFail")
-    public void testCreateHandlerWithEmptyEnvironmentShouldFail() {
+    void constructorThrowsIllegalStateExceptionWhenEnvironmentIsNull() {
         Exception exception = assertThrows(IllegalStateException.class, DynamoDBStreamHandler::new);
         assertThat(exception.getMessage(), containsString(ENVIRONMENT_VARIABLE_NOT_SET));
     }
 
     @Test
-    @DisplayName("testHandlerHandleSimpleRemoveEventWithoutProblem")
-    public void testHandlerHandleSimpleRemoveEventWithoutProblem() throws IOException {
-        DynamodbEvent requestEvent = loadEventFromResourceFile(SAMPLE_REMOVE_EVENT_FILENAME);
-        String response = handler.handleRequest(requestEvent, context);
-        assertNotNull(response);
+    void handlerReturnsSuccessMessageWhenDocumentDoesNotExist() throws IOException {
```

Comment:
I think we should check that the ElasticSearch client is being called. You can use the Mockito expression verify for that

```
verify(restClient, (atLeastOnce()));
verify(restClient, (atMostOnce()));
verify(restClient, (times(1)));  
```